packages/common/object/src/get-name-by-path.ts

```diff
@@ -0,0 +1,35 @@
+import { Any } from '@angular-ru/common/typings';
+import { checkValueIsEmpty } from '@angular-ru/common/utils';
+
+import { getValueByPath } from './get-value-by-path';
+import { isObject } from './is-object';
+
+export function getNameByPath<T>(item: T, displayKey?: keyof T): string {
```

Comment:
didn't catch:
1. what does the word `name` means in the context of `T` which is string or any abstract object? e.g. `{ opened: true, width: 300, title: 'Some Titile' }` — which of these fields can be called a name?
2. why is there a separate function? i think `getValueByPath` is quite enough for these purposes. i see it insures against falsy values, but there is no need for this. anyway you can use `??` in the pipe.

Reply:
1. `name` looks a little confusing, so I renamed pipe: `itemName` -> `displayItem`
2. removed `getNameByPath`